docs/content/querying/filters.md

```diff
@@ -81,13 +86,21 @@ The following matches any dimension values for the dimension `name` between `'ba
 }
 ```
 
-### Extraction filter
+The JavaScript filter supports the use of extraction functions, see [Filtering with Extraction Functions](#filtering-with-extraction-functions) for details.
+
+
+### Extraction filter (Deprecated)
```

Comment:
Could you remove the (Deprecated) and just leave the note? The reason is that altering the title of a section changes the named anchors and can break incoming links.